### ISSUE TITLE
FeatureGroup do not support tooltip with permanent: true (as advertised)

### ISSUE THREAD

**Issue by elmatou** [Leaflet/Leaflet#6938]
- [x] I've looked at the [documentation](http://leafletjs.com/reference.html) to make sure the behavior is documented and expected
- [x] I'm sure this is a Leaflet code issue, not an issue with my own code nor with the framework I'm using (Cordova, Ionic, Angular, React…)
- [x] I've searched through the issues to make sure it's not yet reported

**Steps to reproduce**
Add a featureGroup with `bindToolip` to the map
Add a marker to the featureGroup

**Expected behavior**
every Marker should own (or share in this case) a tooltip which is permanently shown.
as documented "bindPopup binds a popup to all of the layers at once (likewise with bindTooltip)"

**Current behavior**
no Marker is shown
no Tooltip is shown 
in console I get "Error: Unable to get source layer LatLng."

**Environment**
- Leaflet version:
- Browser (with version):
- OS/Platform (with version):

**Additional context**

when removing permanent: true is removed everything works fine.

**Minimal example reproducing the issue**

https://next.plnkr.co/edit/S2W7zcKTkVIrFLur

- [x] this example is as simple as possible
- [x] this example does not rely on any third party code

**Comment by johnd0e** [comment]
> every Marker should own (or share in this case) a tooltip which is permanently shown.

I do not think so. The tooltip should be single.

> as documented "bindPopup binds a popup to all of the layers at once (likewise with bindTooltip)"

Documentation is not precise here, just not best wording.

**Comment by johnd0e** [comment]
> in console I get "Error: Unable to get source layer LatLng."

That is because you are trying to open tooltip for empty FeatureGroup.
Addressed in #7540.

**Comment by johnd0e** [comment]
So here is correct sample:
```js
var markerGroup = new L.FeatureGroup([new L.Marker(map.getCenter())])
  .bindTooltip('balbla Tooltip', {permanent:true})
  .addTo(map);
map
  .on('click', function(e) {
    var marker = new L.Marker(e.latlng)
      .addTo(markerGroup);
  });
```
This keeps single tooltip and that is intended behavior.
